oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/IndexDefinition.java

```diff
@@ -583,10 +609,6 @@ public int getMaxExtractLength() {
         return maxExtractLength;
     }
 
-    public String getScorerProviderName() {
-        return scorerProviderName;
-    }
-
```

Comment:
unused?

Reply:
yes, I have included a number of unrelated code improvements in this separate commit https://github.com/oak-indexing/jackrabbit-oak/pull/508/commits/3fc57cbf83b74987ede84d7e169e3ea1879b8f83